armi/reactor/components/component.py

```diff
@@ -792,8 +791,7 @@ def setDimension(self, key, val, retainLink=False, cold=True):
             If True, the val will be applied to the dimension of linked
             component which indirectly changes this component's dimensions.
         cold : book, optional
```

Comment:
While we're at it, may as well fix the typo "book" to "bool"

Reply:
Responded to both your thoughts.

They were both good ones!